about.html

```diff
@@ -0,0 +1,83 @@
+            <li class="head-link"> Mypage</li>
+            <li class="head-link">Logout</li>
+        </div>
+        <img class="close" alt="close" src="./Images/close.png">
+        <img alt="logo" class="header-logo" src="./Images/logo.png">
+        <li class="links link1"><a class="menu-links" href="./index.html"> Home</a></li>
+        <li class="links link2"><a class="menu-links" href="./about.html">About</a></li>
+        <li class="links link3"><a class="menu-links" href="#program">program</a></li>
+        <li class="links link3"><a class="menu-links" href="#program">Speakers</a></li>
+    </ul>
+    <section class="pg1-sec">
+        <h2 class="hello-world">"Hello, Sharing world"</h2>
+        <h1 class="event-name">The World Games <br>2022 Birmingham, USA</h1>
+        <p class="event-dis">international multi-sport event organized with the support of the International Olympic
+            Committee. Held the year following the Summer Olympic Games, The World Games 2022 will showcase a New
+            Generation of Global Sports in Birmingham from July 7-17, 2022. An anticipated 3,600 elite athletes from
+            over 100 countries will compete for gold in more than 30 of the fastest growing sports in the world. An
+            anticipated 3,600 elite athletes
+            from over 100 countries will compete for gold in more than 30 of the fastest growing sports in the
+            world.</p>
+        <p class="event-dis enjoy">Buy ticket and enjoy</p>
+        <p class="event-start">Birmingham, USA | July 7 - 17, 2022</p>
+    </section>
+    <section class="sec-logo">
+        <h2 class="what-logo">The World Games logo</h2>
+        <hr class="logo-hr">
+        <p class="dis-logo">The logo of World Games submmit 2019 was decided through the logo competition from 9
+            June to 12 Nov</p>
+        <img alt="logo" class="com-logo" src="./Images/logi.png">
+    </section>
+    <hr class="line">
+    <section class="past-story">
+        <h1 class="see-past">See past events about World Games logo</h1>
+        <p class="dis-past">logo of World Games submmit 2019 was decided through the logo competition from 9</p>
+        <div class="red-img">
+            <img class="img1" src="./Images/2.jpg" alt="Evint Image">
+            <img class="img1" src="./Images/1.jpg" alt="Evint Image">
+        </div>
+    </section>
+    <section class="sec-logo partners">
+        <h3 class="what-logo">Partner</h3>
+        <hr class="logo-hr">
+        <div class="partner-logo-container">
+            <img alt="Microsoft logo" class="patner-logo" src="./Images/microsoft.png">
+            <img alt="Coca Cola logo" class="patner2-logo" src="./Images/cocacola.png">
+        </div>
+    </section>
+    <section class="sec-logo last-part">
+        <img alt="logo" class="footer-logo" src="./Images/logo.png">
+        <p class="dis-logo last-p">The World Game <br> Some Rights Reserved.</p>
+    </section>
+
```

Comment:
- [OPTIONAL] The HTML uses semantic tags, sections use `section`, articles use `article`, footer use `footer` tags, ... Kindly use footer tags to write the last section of your page and not divs. `<footer class="sec-logo "></footer>` 👍🏻 I know you can do it 😄 💪🏻 
- [OPTIONAL] Kindly change the background color and the text color of the About Page footer section, you have to stick to the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),

- Your design:

![cap11](https://user-images.githubusercontent.com/79058364/170033693-0ca10dde-cf1a-4b8f-abd8-49443bc19a62.png)



- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![footabou1](https://user-images.githubusercontent.com/79058364/170037105-cdec3174-540d-42d9-96a6-5390be014cae.png)